src/MediaRatings1L.java

```diff
@@ -0,0 +1,178 @@
+import java.util.LinkedList;
+import java.util.Queue;
+
+/**
+ * MediaRatings represented as a java.util.Map with implementations of primary
+ * methods.
+ *
+ * @convention $this.Pair.media() != null and $this.Pair.rating() >= 0 and
+ *             $this.Pair.rating() <= 5 and MediaRating holds the string and
+ *             integer pair which is held in a queue
+ *
+ * @correspondence this = entries($this.rep)
+ *
+ */
+public class MediaRatings1L extends MediaRatingsSecondary {
+
+    /*
+     * Private Members
+     */
+
+    /**
+     * Elements included in {@code this}.
+     */
+    private final class MediaRating {
```

Comment:
You may have heard me recommend this before, but I might recommend using a Java record here. The caveat being that records are immutable, so you if you intend to edit ratings regularly, you would need a way to replace existing ratings. That said, I tend to think the pros far outweigh the cons as records come with equals, toString, and hashCode out of the box.

Reply:
Here's what that would look like for the record:

```suggestion
    private record MediaRating(String media, int rating) { }
```

And if you want to simplify things even more, you could place the MediaRating object in its own file (simplifies need to write `MediaRating1L.MediaRating`. I suspect you might have an interface for this higher up the hierarchy, which is why it's implemented here. Otherwise, I don't think the interfaces will compile.